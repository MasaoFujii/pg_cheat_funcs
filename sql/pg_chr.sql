CREATE EXTENSION pg_cheat_funcs;

\pset null '(null)'

SELECT chr(88) = pg_chr(88);
SELECT chr(12354) = pg_chr(12354);
SELECT chr(57000);
SELECT pg_chr(57000);
SELECT chr(2000000);
SELECT pg_chr(2000000);

DROP EXTENSION pg_cheat_funcs;
