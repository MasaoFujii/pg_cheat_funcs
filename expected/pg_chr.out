CREATE EXTENSION pg_cheat_funcs;
\pset null '(null)'
SELECT chr(88) = pg_chr(88);
 ?column? 
----------
 t
(1 row)

SELECT chr(12354) = pg_chr(12354);
 ?column? 
----------
 t
(1 row)

SELECT chr(57000);
ERROR:  requested character not valid for encoding: 57000
SELECT pg_chr(57000);
 pg_chr 
--------
 (null)
(1 row)

SELECT chr(2000000);
ERROR:  requested character too large for encoding: 2000000
SELECT pg_chr(2000000);
 pg_chr 
--------
 (null)
(1 row)

DROP EXTENSION pg_cheat_funcs;
